app/domain/SdesFile.scala

```diff
@@ -317,3 +325,23 @@ case class PostponedVatStatementFileMetadata(periodStartYear: Int,
                                              fileRole: FileRole,
                                              source: String,
                                              statementRequestId: Option[String]) extends SdesFileMetadata
+
+case class CashAccountStatementFile(filename: String,
+                                    downloadURL: String,
+                                    size: Long,
+                                    metadata: CashAccountStatementFileMetadata,
+                                    eori: String)
+                                   (implicit messages: Messages
+                                   ) extends Ordered[CashAccountStatementFile] with SdesFile {
+
+  val formattedSize: String = Formatters.fileSize(size)
```

Comment:
Could you please add tests for this val, formattedMonth and for the compare method (if possible) ?